src/broker/broker.c

```diff
@@ -480,10 +480,6 @@ int main (int argc, char *argv[])
         goto cleanup;
     }
 
-
-    /* Initialize module infrastructure.
-     */
-
```

Comment:
oops, was this supposed to go into the commit prior to this one?

Reply:
Aw dang it!  Good catch.  Will fix.